<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmKit</class>
 <widget class="QDialog" name="FrmKit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de Kits de producto.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Código:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtCodigo_kit">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="txtDesc_kit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>PVP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="txtPvp_kit">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>0,00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tabla">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">alternate-background-color: rgb(255, 255, 255);
font: 8pt &quot;Sans&quot;;
background-color: rgb(248, 245, 194);</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 127);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Filtro:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="4">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Dto:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="6">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnGuardar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Guardar</string>
          </property>
          <property name="icon">
           <iconset resource="../maya.qrc">
            <normaloff>:/Icons/PNG/Save.png</normaloff>:/Icons/PNG/Save.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDeshacer">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Deshacer</string>
          </property>
          <property name="icon">
           <iconset resource="../maya.qrc">
            <normaloff>:/Icons/PNG/undo.png</normaloff>:/Icons/PNG/undo.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Cerrar</string>
          </property>
          <property name="icon">
           <iconset resource="../maya.qrc">
            <normaloff>:/Icons/PNG/Exit.png</normaloff>:/Icons/PNG/Exit.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Código:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="6">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnAnadir">
          <property name="text">
           <string>Añadir producto</string>
          </property>
          <property name="icon">
           <iconset resource="../maya.qrc">
            <normaloff>:/Icons/PNG/add.png</normaloff>:/Icons/PNG/add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnQuitar">
          <property name="text">
           <string>Quitar producto</string>
          </property>
          <property name="icon">
           <iconset resource="../maya.qrc">
            <normaloff>:/Icons/PNG/borrar.png</normaloff>:/Icons/PNG/borrar.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1" colspan="5">
       <widget class="QComboBox" name="cboOrden"/>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QCheckBox" name="chkStock">
        <property name="text">
         <string>Quitar de stock</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="5">
       <widget class="QLineEdit" name="txtBuscar">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Cantidad:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="5">
       <widget class="QDoubleSpinBox" name="spinPVP">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <double>-10000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QTableView" name="tabla_buscar">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(191, 255, 191);
alternate-background-color: rgb(222, 255, 222);</string>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QLineEdit" name="txtCodigo">
        <property name="minimumSize">
         <size>
          <width>139</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QSpinBox" name="spinCantidad">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="10" column="5">
       <widget class="QLineEdit" name="txtDto">
        <property name="minimumSize">
         <size>
          <width>73</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>126</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>0,00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Descripción:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="5">
       <widget class="QLineEdit" name="txtDescripcion">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>PVP:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Buscar por:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Sentido:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QComboBox" name="cboSentido"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnAnadir</tabstop>
  <tabstop>btnQuitar</tabstop>
  <tabstop>tabla_buscar</tabstop>
  <tabstop>txtCodigo</tabstop>
  <tabstop>spinPVP</tabstop>
  <tabstop>txtDescripcion</tabstop>
  <tabstop>spinCantidad</tabstop>
  <tabstop>txtDto</tabstop>
  <tabstop>chkStock</tabstop>
  <tabstop>btnGuardar</tabstop>
  <tabstop>btnDeshacer</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>txtCodigo_kit</tabstop>
  <tabstop>tabla</tabstop>
  <tabstop>txtDesc_kit</tabstop>
  <tabstop>txtPvp_kit</tabstop>
 </tabstops>
 <resources>
  <include location="../maya.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>FrmKit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>868</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
